src/javascript/DesignSystem/ContentTable/ContentTableHeader/ContentTableHeader.jsx

```diff
@@ -11,7 +10,7 @@ const ContentTableHeader = ({columns, order, orderBy}) => {
         <TableHead>
             <TableRow>
                 <TableCell padding="checkbox">
-                    <Checkbox indeterminate/>
+                    {/* Checkbox location */}
```

Comment:
Why removing this element ? Maybe add a TODO ?

Reply:
yes, a TODO is better, thank you